src/features/Licenses/LicensesTable/columns.jsx

```diff
@@ -15,6 +15,13 @@ const columns = [
     Header: 'Enrolled',
     accessor: 'numberOfStudents',
   },
+  {
+    Header: 'Pending',
+    accessor: 'pendingStudents',
+    Cell: ({ row: { purchasedSeats = 0, numberOfStudents = 0, numberOfPendingStudents = 0 } }) => (
+      <span>{purchasedSeats - numberOfStudents - numberOfPendingStudents}</span>
+    ),
+  },
   {
     Header: 'Remaining',
```

Comment:
Remaining column should be the calculation of purchasedSeats - numberOfStudents - numberOfPendingStudents